server/core/config.py

```diff
@@ -3,11 +3,14 @@
 from pydantic import BaseSettings
 
 API_V1_STR = "/api/v1"
+COASTLINES_V1_STR = API_V1_STR + "coastlines"
 
 
 class Settings(BaseSettings):
     DATABASE_URL: str
-
+    API_V1_ROUTE: str = API_V1_STR
+    COASTLINES_V1_ROUTE: str = COASTLINES_V1_STR
```

Comment:
This is a great idea. It's one I haven't considered... My original intention with this class was to have this closely coupled with the system's environment variables. 

What do you think about separating API routes to a constants file instead?

I don't have a strong opinion on whether we should change your approach or not. I only have one essential comment/suggestion for us:

- Is there a better structure to manage our API route constants? (e.g., dicts, Enum, or a separate constants file)?

Reply:
I think that a separate constants file would be useful but we'd then need to create a utils folder. I'm not sure if it'll be necessary to do it as of now since we can still manage everything in the config py file, but down the road as the code base becomes bigger, it'd definitely be an avenue to pursue.